keps/sig-node/2625-cpumanager-policies-thread-placement/README.md

```diff
@@ -183,80 +199,31 @@ With `threads_per_cpu` is typical 2 on x86_64 with SMT enabled - but this number
 In order to make the resource reporting consistent, and avoiding cascading changes in the system, we enforce the request constraints at admission time.
 This approach follows what the Topology Manager already does.
 
-### Alternatives
```

Comment:
Why remove the alternatives? I think we should keep them for future reference and context.